package/libs/zlib/Makefile

```diff
@@ -74,7 +56,7 @@ define Build/InstallDev
 	$(CP)	$(PKG_INSTALL_DIR)/usr/lib/libz.{a,so*} \
 		$(1)/usr/lib/
 	mkdir -p $(1)/usr/lib/pkgconfig
-	$(CP)	$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/zlib.pc \
+	$(CP)	$(PKG_INSTALL_DIR)/usr/share/pkgconfig/zlib.pc \
```

Comment:
Can you elaborate on this change?  pkgconfig files are normally put in `/usr/lib/pkgconfig`.

Reply:
cmake defaults to share instead of lib

Reply:
I see.  It's written in the CMakeLists.txt file.  Thanks